packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationSteps/YppAuthorizationSyncStep/YppAuthorizationSyncStep.tsx

```diff
@@ -71,6 +73,12 @@ export const YppAuthorizationSyncStep = () => {
             )}
           />
         </FormField>
+      ) : (
+        <Banner
+          title="Only auto-synced videos are rewarded"
+          description="Manually uploaded videos are not qualifying for programme rewards. To get rewards for content upload consider switching to syncing Youtube videos."
```

Comment:
```suggestion
          description="Manually uploaded videos are not qualifying for program rewards. To get rewards for content upload consider switching to syncing YouTube videos."
```
Let's keep the wording consistent across the app

Reply:
@dmtrjsg let's solve this one as I think we're using a word "programme" in the config atm. I'll change the config afterwards